hipcheck/src/plugin/download_manifest.rs

```diff
@@ -277,95 +264,6 @@ impl ParseKdlNode for DownloadManifestEntry {
 	}
 }
 
-impl DownloadManifestEntry {
-	/// This function does the following:
-	/// 1. Download specified plugin
-	/// 1. Verify its size and hash
-	/// 1. Extract plugin into plugin-specific folder
-	/// 1. Finds `plugin.kdl` inside plugin-specific folder and calls this recursively
-	pub fn download_and_unpack_plugin<'a>(
-		&self,
```

Comment:
Do I understand correctly that all the deleted code in this file is superceded by the additions in `retrieval.rs`?

Reply:
Yes all of the logic has moved to `retrieval.rs`